### ISSUE TITLE
Re-enable LWT to static themes updates for Android

### ISSUE THREAD

**Issue by buttercookie42** [mozilla/addons-server#9941]
Once [bug 1429488](https://bugzilla.mozilla.org/show_bug.cgi?id=1429488) lands, the current level of theme support for LWTs in Android (header image and accent colour) will work with static themes, too.

This means that it should then be safe to serve updates to migrated LWTs to Android users as well, and also allow normal access to static themes (although ideally we'd suppress themes without a header image for Android users, because those will remain somewhat pointless for now - no idea whether that's easily possible, though).

**Comment by nt1m** [comment]
[Bug 1429488](https://bugzilla.mozilla.org/show_bug.cgi?id=1429488) landed on mozilla-central.

**Comment by eviljeff** [comment]
When will it be available to release users?

**Comment by buttercookie42** [comment]
Mid-December if I can still uplift, February (Firefox 65) otherwise.

**Comment by eviljeff** [comment]
we'll have to do #9967 so we're exposing them to appropriate versions of Firefox for Android, but that's kinda blocked on knowing what the minimum version is (64 or 65)

**Comment by AlexandraMoga** [comment]
I've highlighted to @eviljeff that LWT updates (manual checks or background) were not received on -dev after the lwt themes used for testing were migrated. 

Given that the functionality can't be verified on -dev yet, I'm suggesting to postpone the push in prod until next week, provided that -dev and stage can be properly verified until then.

**Comment by eviljeff** [comment]
There's a problem with the LWT theme version check (#10924).  It _may_ be related to the LWT -> static theme upgrade code or may be a coincidence but we're going to revert it as it's untestable at least.

**Comment by AlexandraMoga** [comment]
@eviljeff I had a couple of LWTs from my previous testing installed on AMO -dev, both on Desktop and Android, for which I've checked this morning if the updates to ST were received.

On desktop the LWTs were updated but on Android, I'm still having the LWTs. I'm thinking to give it a little more time, maybe the update check hasn't run on mobile yet.

I'd also like to verify the update checks on stage, so if you can share some other LWT Ids which were migrated (like you did in https://github.com/mozilla/addons-server/issues/10924#issuecomment-473989485) that would be great.

**Comment by eviljeff** [comment]
here are some from stage (I've not checked all the ids are still okay)
```
+----------------------+-----------------+
| lightweight_theme_id | static_theme_id |
+----------------------+-----------------+
|                15105 |          620573 |
|                15316 |          620574 |
|                15424 |          620575 |
|                15533 |          620576 |
|                15207 |          620577 |
|                15106 |          620578 |
|                15317 |          620579 |
|                15425 |          620580 |
|                15534 |          620581 |
|                15208 |          620582 |
|                15107 |          620583 |
|                15318 |          620584 |
|                15426 |          620585 |
|                15535 |          620586 |
|                15209 |          620587 |
|                15108 |          620588 |
|                15320 |          620589 |
|                15427 |          620590 |
|                15536 |          620591 |
|                15210 |          620592 |
+----------------------+-----------------+
```

**Comment by AlexandraMoga** [comment]
@eviljeff, thanks. I've managed to work with some of those ids and I can confirm that `/themes/update-check/` is working on stage as well, but only on Desktop so far. 

On Android (AMO -dev), I still haven't received any updates. I'm currently trying to set up the browser console on mobile to see if theme versioncheck is actually running.

**Comment by AlexandraMoga** [comment]
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1537723 for the `/themes/update-check/` issue on Android

**Comment by AlexandraMoga** [comment]
Look like https://bugzilla.mozilla.org/show_bug.cgi?id=1537723 has been fixed in Beta 67.0b7

We can mark this issue as verified fixed as well.

**Comment by eviljeff** [comment]
@AlexandraMoga do you think we're safe removing the last pieces of support for LWT on AMO while this fix is not on release?  Do QA typically re-test on release or was the test on beta the end of it?  

(Once we delete all the LWT from dev and delete the supporting models it won't be possible to install an LWT to test the upgrade path, unless the tester already has a LWT installed previously)

**Comment by AlexandraMoga** [comment]
@eviljeff normally Firefox QA doesn't verify a fix in release unless it is an uplift. I don't think this will apply to https://bugzilla.mozilla.org/show_bug.cgi?id=1537723 - I haven't seen any intention to uplift it to the current release (FF66) - so, in theory, the test cycle has ended.

However, in the spirit of being prepared for the unexpected, maybe we can keep LWT support for a little longer, if that doesn't impact your projects too much. Consider that we still have more than one month until 67 becomes the next release.
